src/extensions/default/Phoenix-markdown-toolbar/dialogs.js

```diff
@@ -1,3 +1,23 @@
+/*
+ * GNU AGPL-3.0 License
+ *
+ * Modified Work Copyright (c) 2021 - present core.ai . All rights reserved.
+ * Original work Copyright (c) 2012 - 2021 Adobe Systems Incorporated. All rights reserved.
```

Comment:
Misattribution in copyright strings. The Copyright should go to the original author of the extension instead of Adobe.

Reply:
changes made in copyright